### DISCUSSION TITLE
Infoboxes columns

### DISCUSSION THREAD

**Discussion by PrelawGunmir** [SlRvb/Obsidian--ITS-Theme#323] [Q&A]
I am using obsidian as a database to store information about my story, such as characters, the world, and other terminology. When I try to make an infobox with the wikipedia styling, I can only make it where the "name" of the data is above, and below is the value I am inputting. I am using obsidian as a database to store information about my story, such as characters, the world, and other terminology. For example, if I were to put a character's birthday in the infobox, it would have to say Birthday above and the date below. Is there a way I can change this to where the name is on the left and value on the right, like wikipedia usaully does?
What I get:
![image](https://github.com/user-attachments/assets/333d470c-cae8-4829-8cf8-444df79ae813)
What I want (or as close as possible):
![image](https://github.com/user-attachments/assets/fb8c666d-afd8-4912-8855-e77f174fa9f3)

**Comment by PrelawGunmir** [comment]
I have found a temporary fix, but I will use this method if it is the only way to get it to display how I want.
![image](https://github.com/user-attachments/assets/c215adc0-12bb-425a-afde-545fccf9df19)
![image](https://github.com/user-attachments/assets/14840a05-218e-4f1a-87d8-34175648144d)

**Comment by SlRvb** [comment]
You probably just want to use the table like:
```
> [!infobox]
> # Kael Ardin
> 
> | | |
> |---|---|
> | Birthday | March 11th |
```
